### PR TITLE
Add --storage as CLI argument

### PR DESCRIPTION
**Pull request by MattWellie**
Adds optional command line argument `--storage`. If specified will request attached storage for the job.

### CODE REVIEW

File: server/main.py [hunk 1]
```diff
@@ -85,6 +85,10 @@ async def index(request):
     image = params.get('image') or DRIVER_IMAGE
     cpu = params.get('cpu', 1)
     memory = params.get('memory', '1G')
+
+    # defaults to no attached storage in Driver jobs
+    storage = params.get('cpu', None)
```
Comment (illusional) [inline server/main.py:90]:
```suggestion
    storage = params.get('storage', None)
```

File: server/main.py [hunk 2]
```diff
@@ -177,6 +181,8 @@ async def index(request):
     job.image(image)
     if cpu:
         job.cpu(cpu)
+    if storage:
+        job.cpu(storage)
```
Comment (illusional) [inline server/main.py:185]:
```suggestion
        job.storage(storage)
```

**Review by illusional — APPROVED** [review]
I'm going to apply my suggestion feedback straight away haha, otherwise this looks good.

File: server/main.py (hunk 1 above)
Comment (MattWellie) [reply server/main.py]: doiii

File: server/main.py (hunk 2 above)
Comment (MattWellie) [reply server/main.py]: DOIIII

**Pull request merged**